test/common/schemaform/widgets/ArrayCountWidget.unit.spec.jsx

```diff
@@ -133,6 +133,50 @@ describe('Schemaform <ArrayCountWidget>', () => {
     });
     expect(onChange.firstCall.args[0]).to.equal(undefined);
   });
+  it('should handle change that removes with offset', () => {
+    const onChange = sinon.spy();
+    const tree = SkinDeep.shallowRender(
+      <ArrayCountWidget
+          id="1"
+          value={[{}, {}]}
+          schema={{ type: 'string' }}
+          required
+          disabled={false}
+          formContext={{}}
+          onChange={onChange}
+          options={{
+            countOffset: -2
+          }}/>
+    );
+    tree.subTree('input').props.onChange({
+      target: {
+        value: '1'
+      }
+    });
+    expect(onChange.firstCall.args[0]).to.equal(undefined);
```

Comment:
Start with 2 values, offset -2
Change down to 1 value, offset -2
Expect `undefined` to be returned because the number of values < number of offset.

Got it. The description wasn't super descriptive, but I think it makes sense after reading it. Not sure what a better description would be, though.